lib/fileio/copy.go

```diff
@@ -322,6 +323,15 @@ func mkdirAll(dst string, mode os.FileMode, uid, gid int, preserveParentOwner bo
 		}
 		prevDir = absDir
 	}
+
```

Comment:
I add a isDir field in the function. so when we copy dirs, the target dir could maintain the src ownership according to the boolean value.